k8s/ithc/common-overlay/ia/kustomization.yaml

```diff
@@ -12,7 +12,8 @@ patchesStrategicMerge:
 - ../../../namespaces/ia/ia-case-payments-api/ithc.yaml
 - ../../../namespaces/ia/ia-bail-case-api/ithc.yaml
 - ../../../namespaces/ia/ia-case-notifications-api/ithc.yaml
-- ../../../namespaces/ia/ia-case-documents-api/ithc.yaml
```

Comment:
Why removal of` ia-case-documents-api` ?

Reply:
discussed with Rajesh, duplicate entry

Reply:
its a duplicate entry